Tests/GRPCCoreTests/Call/Client/ClientRequestTests.swift

```diff
@@ -19,10 +19,13 @@ import XCTest
 final class ClientRequestTests: XCTestCase {
   func testSingleToStreamConversion() async throws {
     let (messages, continuation) = AsyncStream.makeStream(of: String.self)
-    let single = ClientRequest.Single(message: "foo", metadata: ["bar": "baz"])
+    let single = ClientRequest.Single(
```

Comment:
I think adding the `ExpressibleByString{Interpolation,Literal}` conformance to `MetadataValue` will mean all of these changes to tests aren't required.